### DISCUSSION TITLE
Firefly Shimmer 2.0.0

### DISCUSSION THREAD

**Discussion by nicole-obrien** [iotaledger/firefly#4740] [Announcements]
The first production release for Firefly Shimmer (Firefly desktop v2.0.0) 🚀 

### Change log
------
### File Hashes
[How to verify the authenticity of your Firefly Desktop download](https://wiki.iota.org/introduction/how_tos/verify_firefly_download#verify-your-firefly-desktop-download)
| File | Platform | SHA256 Hash |
| --- | --- | --- |
| firefly-desktop-2.0.0.exe | Windows | 87ab2d173ec8806f5424e239b4a80de52a464914108b56db79438a191842f06c |
| firefly-desktop-2.0.0.AppImage | Linux | 1035c75c36163c77c23998bb306b3ec78348075bd044ddf6b62d775c14620e32 |
| firefly-desktop-2.0.0.dmg | MacOS |  8c165b8a1973432cfb746d0c458e243f184ce15b696af86a288e30d55c1c7174 |

<hr /><em>This discussion was created from the release <a href='https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0'>Firefly Shimmer 2.0.0</a>.</em>

**Comment by Riddle1988** [comment]
Hi,
Thanks for the new release it looks good, but I have a versioning question.

It seems this release was triggered with a pre-release flag, but in the comment above it is stated that this is the first production release for Shimmer. Was this done on purpose? 
I guess you guys are using semantic versioning, so that is why we have a bump to 2.0, but since it is not beta anymore I was expecting to see a new release, not a pre-release (but I do not know your release workflow).

Consequences of this are that in the software itself we can see the text "You are running a pre-release version of Firefly" when going into the check for updates menu.

Sorry, if this is the wrong place to ask, but I didn't want to open an issue since I do not know your workflow :)

**Reply by Tuditi** [reply]
It's a pre-release, since we want to keep the official Firefly IOTA version as the production release. I hope this answers your question. :relaxed:

**Comment by mishtavince** [comment]
Hello, 
Downloaded the MAC version. I Get a blanc screen while opening it. 
Attached images.
![1](https://user-images.githubusercontent.com/34732630/192809798-598ccab9-e967-4ddc-9598-9aac7524139f.jpg)
![2](https://user-images.githubusercontent.com/34732630/192809820-a9ae6322-ab98-45ac-828e-bab44e66b72a.jpg)

**Reply by gemramir** [reply]
The same happened to me. No chance to get anywhere else.

**Reply by Riddle1988** [reply]
Maybe because of this?

https://github.com/iotaledger/firefly/issues/4750#issuecomment-1261487831

**Reply by mishtavince** [reply]
That was for a windows machine. Wonder what the fix is for a Mac..

**Reply by Tuditi** [reply]
We're looking into this. One of the possible reasons is that we upgraded the machines on which Firefly is built to newer OS versions. This means that the C/C++ runtime libraries of older OS versions are not supported.

**Reply by Schmagauke** [reply]
That's it - High Sierra is not supported - you have to upgrade your MacOS to Catalina or higher

**Comment by SF52Akurey** [comment]
hello, this means that in the end the IOTA firefly will integrate shimmer and other tokens? do you know when a new official version will be out?
thank you

**Reply by Tuditi** [reply]
No, we will integrate IOTA into Firefly Shimmer (let's call this v2). Which at that point will replace the current IOTA Firefly (let's call it v1) completely. At that point in time, Firefly v1 will be deprecated.

**Comment by 55thedac** [comment]
Am I the only one who keeps getting this error?  ***We received an invalid amount, please try again***
I couldn't verify  wallet, where am I doing wrong?

**Reply by Tuditi** [reply]
Please open a Github bug report [here](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug+report&template=bug_report.yml&title=%5BBug+report%5D%3A+). It would help a lot if you could describe the exact reproduction steps as well.

**Comment by safi1972** [comment]
I can't log into the app. It keeps saying "something went wrong" What should I do?

**Comment by safi1972** [comment]
I can't log into the app. It keeps saying "something went wrong" What should I do?
[1.docx](https://github.com/iotaledger/firefly/files/9792332/1.docx)

**Reply by Tuditi** [reply]
Could you send us an error log? You can find it from the top bar ('Firefly Shimmer' > Error Log)

**Comment by tristanc39** [comment]
I have installed Firefly Shimmer 2.0.0 pre release, when I go to create a profile it only lets me use a custom network, I add the node manually (https://api.testnet.shimmer.network) but firefly shimmer does not connect to it.
Does it only work locally ?

**Reply by Tuditi** [reply]
Normally, it should work fine. Can you try using 2.0.1 (https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.1) and/or 2.1.0-alpha 3 (https://github.com/iotaledger/firefly/releases/tag/desktop-2.1.0-alpha-3). These versions include the latest fixes.
